src/Repository/Geo/ZoneRepository.php

```diff
@@ -500,4 +514,19 @@ public function searchCitiesInZones(array $zones, string $search): array
 
         return $qb->getQuery()->getResult();
     }
+
+    public function findAllDepartmentSiteIndexByCode(): array
```

Comment:
what about moving this into DepartmentRepository?

Reply:
we don't have a custom department like 64PB or FDE in the departmental table